src/Fixer/ControlStructure/NoAlternativeSyntaxFixer.php

```diff
@@ -67,7 +67,7 @@ public function isCandidate(Tokens $tokens)
      */
     public function getPriority()
     {
-        return 1;
+        return 35;
```

Comment:
why that change ?

Reply:
and for other changes...

Reply:
> AlignMultilineComment should run before every PhpdocFixer

> `AlignMultilineCommentFixer` needed to change from -40 to +30

would adjusting priority of `PhpdocFixer` instead of `AlignMultilineCommentFixer` produce less priority changes ?

Reply:
Unfortunately, no. If it would've, I'd have done that instead 😉

You can see that in the graph as well.